service/web_controller.rb

```diff
@@ -71,7 +71,7 @@ def as_json(*args)
   measure_dto = request.params # Uses the Rack::JSONBodyParser middleware
 
   # Prepare CQM Measure
-  madie_measure = JSON.parse(measure_dto["measure"])
+  madie_measure = JSON.parse(measure_dto["measure"],:max_nesting => false)
```

Comment:
do we know what property/field it is failing on? if yes, we can probably set a number that can be max for that property.

Reply:
@adongare No we don't, I did set it to 150 and it worked for this measure, but I guess we probably shouldn't make it work for just this measure? That's why I changed the max number to setting the nesting to false. What do you think?

Reply:
I don't have a strong opinion. It might be okay I guess. but this becomes a candidate for DoS attacks. @riddhi-desai / @sb-benohe  might have ideas on this. I remember in Bonnie we had a similar issue while storing documents in the database. there we set the max depth to 500

Reply:
@adongare Ok I see your point. I can check the depth and set it to a number. We can also discuss this at the meeting. Thanks so much Ashok!!!

Reply:
np

Reply:
@adongare I set max to 512 as Ben suggested, thanks again!

Reply:
This is changed to: max_nesting: 512 so resolving this conversation.